src/main/java/hometask3/shape/Round.java

```diff
@@ -0,0 +1,20 @@
+package hometask3.shape;
+
+abstract class Round implements Shape {
+
+    int radius;
```

Comment:
Round class shouldn't have three parameters. by default - width and height is enough for all cases (circle and oval) and then in the Circle class you will have another constructor with Radius, while Oval class uses constructor from its parent/super class